src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -1239,19 +1241,45 @@ def load_mediation_factors(*_: str) -> pd.Series:
     return mf.set_index(["risk_name", "mediator_name", "affected_entity"])
 
 
-def load_hf_deltas(*_: str) -> pd.Series:
+def load_hf_deltas(_: str, location) -> pd.Series:
     deltas = pd.read_csv(paths.FILEPATHS.HEART_FAILURE_MEDIATION_DELTAS)
     deltas = deltas.set_index(["age_start", "sex"])[
         [c for c in deltas.columns if c.startswith("draw_")]
     ]
-    # # Get the full index
+    # Get the full index
     idx = (
-        load_population_structure(data_keys.POPULATION.STRUCTURE, "Alabama")
+        load_population_structure(data_keys.POPULATION.STRUCTURE, location)
```

Comment:
The changes here in `load_hf_deltas` are what we chatted about in mid-day. I changed from just using "Alabama" pop structure to `location`'s and also sorted the index below before forward- and back-filling missing data. This is more proper, but I think the risk that this actually changes any already-existing artifacts is very, very small and so I won't be re-running v&v.